File: JavaScript_Courses/JavaScript_Basics/Preparation/04_For_Loop_Exercise/04_Clever_Lily.js
function cleverLily(input) {
    let age = Number(input[0]);
    let washingMachinePrice = Number(input[1]);
    let toyPrice = Number(input[2]);

    let toyCount = 0;

    let totalMoney = 0;
    let giftMoney = 10;

    for (let i = 1; i <= age; i++) {
        if (i % 2 == 1) {
            toyCount++; // toyCount += 1;
        } else {
            // totalMoney += i * 5;
            totalMoney += giftMoney;
            totalMoney--;
            giftMoney += 10;
        }
    }

    totalMoney += toyCount * toyPrice;

    if (totalMoney >= washingMachinePrice) {
        console.log(`Yes! ${(totalMoney - washingMachinePrice).toFixed(2)}`);
    } else {
        console.log(`No! ${(washingMachinePrice - totalMoney).toFixed(2)}`);
    }
}

// cleverLily(["10", "170.00", "6"]);

File: Python_Courses/Python_Basics/Python_PB_2023_04/04_For_Loop_Exercise/04_Clever_Lily.py
age = int(input())
washing_machine_price = float(input())
toy_price = int(input())

total_money = 0
# toy_count = 0
for birthday in range(1, age + 1):  # [1, 2, 3, ..., 9, 10]
    if birthday % 2 == 0:
        total_money += birthday * 10 / 2
        total_money -= 1
    else:
        total_money += toy_price  # toy_count += 1

# total_money += toy_count * toy_price
if total_money >= washing_machine_price:
    print(f"Yes! {total_money - washing_machine_price :.2f}")
else:
    print(f"No! {washing_machine_price - total_money :.2f}")
